src/Microsoft.Performance.Toolkit.Engine.Tests/ToolkitEngineTests.cs

```diff
@@ -582,21 +583,8 @@ public void BuildTable_InternalTables(EngineBuildTableTestCaseDto testCase)
                 Assert.AreEqual(5, builtTable1.Columns.ElementAt(i).Project(0));
             }
 
-            // This table was not explicitly enabled, but it's an metadata (internal) table and should be available
-            var builtTable2 = result.BuildTable(Source5MetadataTable2.TableDescriptor);
-
-            Assert.AreEqual(5, builtTable2.RowCount);
-            Assert.AreEqual(2, builtTable2.Columns.Count());
-            Assert.AreEqual(0, builtTable2.BuiltInTableConfigurations.Count());
-            Assert.AreEqual(0, builtTable2.TableCommands.Count());
-            Assert.IsNull(builtTable2.DefaultConfiguration);
-            Assert.IsNull(builtTable2.TableRowDetailsGenerator);
-
-            for (int i = 0; i < builtTable2.RowCount; i++)
-            {
-                Assert.AreEqual(i + 1, builtTable2.Columns.ElementAt(0).Project(i));
-                Assert.AreEqual($"Source5Row{i + 1}", builtTable2.Columns.ElementAt(1).Project(i));
-            }
+            // Metadata tables are no longer enabled by default and this should fail as it was not explicitly enabled
```

Comment:
>// Metadata tables are no longer enabled by default and this should fail as it was not explicitly enabled [](http://example.com/codeflow?start=12&length=105)

Do we have an explicit enable test for metadata table?

Reply:
added